lib/kramdown_rpf/kramdown.rb

```diff
@@ -97,6 +103,11 @@ def convert_hints(_el, _opts)
         raise NotImplementedError
       end
 
+      # Convert :convert_knowledge_quiz_question -> Markdown (not implemented)
+      def convert_knowledge_quiz_question(_el, _opts)
```

Comment:
If you don't care about the options, you can use `*` as a glob.
```suggestion
      def convert_knowledge_quiz_question(*)
```